<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LGDN14993-W10" ignored="0" name="C:/Users/SATTDN20.03.08/Desktop/myProject/selenium/Selenium/SeleniumTest1/Testcases/RailWay/railways" tests="14" failures="2" timestamp="01 Feb 2021 01:51:56 GMT" time="243.96" errors="0">
  <testcase name="TC01" time="5.189" classname="Testcases.RailWay.railways.TC01_LoginWithValidAccount"/>
  <testcase name="TC02" time="3.612" classname="Testcases.RailWay.railways.TC02_LoginWithBlankUsername"/>
  <testcase name="TC03" time="3.682" classname="Testcases.RailWay.railways.TC03_LoginWithInvalidPassword"/>
  <testcase name="TC04" time="1.312" classname="Testcases.RailWay.railways.TC04_LoginFromBookTicket"/>
  <testcase name="TC05" time="8.669" classname="Testcases.RailWay.railways.TC05_LoginSeveralTimes">
    <failure type="java.lang.AssertionError" message="expected [You have used 4 out of 5 login attempts. After all 5 have been used, you will be unable to login for 15 minutes.] but found [Invalid username or password. Please try again.]">
      <![CDATA[java.lang.AssertionError: expected [You have used 4 out of 5 login attempts. After all 5 have been used, you will be unable to login for 15 minutes.] but found [Invalid username or password. Please try again.]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at Testcases.RailWay.railways.TC05_LoginSeveralTimes.TC05(TC05_LoginSeveralTimes.java:31)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
    </failure>
  </testcase> <!-- TC05 -->
  <testcase name="TC06" time="6.217" classname="Testcases.RailWay.railways.TC06_HomeLogged"/>
  <testcase name="TC07" time="3.281" classname="Testcases.RailWay.railways.TC07_CreateAccount"/>
  <testcase name="TC08" time="3.951" classname="Testcases.RailWay.railways.TC08_LoginWithNotActivatedAccount">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at Testcases.RailWay.railways.TC08_LoginWithNotActivatedAccount.TC08(TC08_LoginWithNotActivatedAccount.java:47)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
    </failure>
  </testcase> <!-- TC08 -->
  <testcase name="TC09" time="7.177" classname="Testcases.RailWay.railways.TC09_ChangeNewPassword"/>
  <testcase name="TC10" time="3.226" classname="Testcases.RailWay.railways.TC10_RegisterWithInvalidAccount"/>
  <testcase name="TC11" time="3.294" classname="Testcases.RailWay.railways.TC11_RegisterWithInvalidAccount"/>
  <testcase name="TC14" time="12.367" classname="Testcases.RailWay.railways.TC14_BookTicket"/>
  <testcase name="TC15" time="5.668" classname="Testcases.RailWay.railways.TC15_BookTicketFrTrainTimetable"/>
  <testcase name="TC16" time="12.056" classname="Testcases.RailWay.railways.TC16_CancelTicket"/>
</testsuite> <!-- C:/Users/SATTDN20.03.08/Desktop/myProject/selenium/Selenium/SeleniumTest1/Testcases/RailWay/railways -->
